### ISSUE TITLE
Root Block Device Potential Breaking Change

### ISSUE THREAD

**Issue by grubernaut** [hashicorp/terraform#12379]
A potentially breaking change was released in Terraform `v0.8.6` from (#11619).

Since #1445 (v0.7.7) the following issue (#3941) has been open, where modifying attributes inside of a `root_block_device` configuration would no longer update the schema values in the statefile. 

This has the potential to form a breaking change for any infrastructure that was created with Terraform v0.7.7 or older, with a direct upgrade to Terraform v0.8.6 or newer. 

Potential break:
```hcl
# terraform <v0.7.7
root_block_device {
  volume_size = 10
  delete_on_termination = false
}
```

Terraform will create the root block device, with `delete_on_terminate` correctly set.

```hcl
# Terraform v0.7.7
root_block_device {
  volume_size = 10
}
```

Terraform will update the statefile to show `delete_on_terminate=true`, but will silently fail to actually update the `root_block_device` in AWS. 

```hcl
# Terraform v0.8.6
root_block_device {
  volume_size = 10
}
```

Upon running an `apply` for the latest config with the latest version of Terraform, a diff will not show for `delete_on_terminate`, but the AWS API will actually be updated, whereas previously the root_block_device would not be updated. 

This can, of course, cause fairly unexpected changes. 

tl;dr: The bugfix from (#11619) allows `root_block_device` parameters to actually be updated in the AWS Console, potentially causing an unknown diff between a users statefile and reality, for any infrastructures managed with Terraform `<v0.7.7`, and updating to `>v0.8.6`
